docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CustomMappingSample.cs

```diff
@@ -1,4 +1,7 @@
 ﻿using System;
+using System.Collections.Generic;
+using Microsoft.ML.Transforms;
+
 namespace Microsoft.ML.Samples.Dynamic
```

Comment:
>Microsoft.ML [](start = 10, length = 12)

we moved to a pattern of adding a "using Microsoft.ML"  , instead of adding  it in the namespace

Reply:
see [https://github.com/dotnet/machinelearning/issues/3205](https://github.com/dotnet/machinelearning/issues/3205)

---
In reply to: [272664635](https://github.com/dotnet/machinelearning/pull/3216#discussion_r272664635) [](ancestors = 272664635)

Reply:
Thank you!


---
In reply to: [272759050](https://github.com/dotnet/machinelearning/pull/3216#discussion_r272759050) [](ancestors = 272759050,272664635)